.github/workflows/test-integration-vizro-ai.yml

```diff
@@ -116,3 +113,33 @@ jobs:
           name: Report-${{ matrix.python-version }}-${{ matrix.label }}
           path: |
             /home/runner/work/vizro/vizro/vizro-ai/tests/integration/reports/report*.csv
+
+  test-integration-vizro-ai-report:
+    needs: test-integration-vizro-ai
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Download All Artifacts
+        uses: actions/download-artifact@v4
+
+      - name: Set current date as env variable
+        id: date
+        run: |
+          echo "::set-output name=date::$(date +'%Y-%m-%dT%H-%M-%S')"
+          echo "TIME_NOW=$(date +'%Y-%m-%dT%H-%M-%S')" >> $GITHUB_ENV
+
+      - name: Report artifacts
+        uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: Report-aggregated-${{ steps.date.outputs.date }}
+          path: |
+            /home/runner/work/vizro/vizro/Report-*
+
+      - name: Upload reports to artifactory
```

Comment:
Managed to do this

<img width="863" alt="image" src="https://github.com/user-attachments/assets/575dbba3-452e-4a39-be8c-7daca3aed502">

Reply:
I didn't dig into the details of the schema, but I assume we can add new "scores" easily in the future, e.g. a layout score, and whatever comes to mind?

This would make the table wider, and previous rows would just have no entry there?

@lingyielia you're the expert DE, how can we ensure future proofing in a way we don't waste old data

Could we also give every prompt not just a tier, but also a unique identifier, so that we can track if we ever changed that prompt, e.g. promptID = 1 (and then count upwards) as the most simple idea

Reply:
schema change is always tricky but always happen overtime. The bottom line is, don't change the existing column names. (e.g., changing from "total_score" to "total_test_score", but referring the same data). Then it's much less painful to deal with schema change (adding or removing columns).